test/Makefile

```diff
@@ -81,6 +81,7 @@ endif
 include ../src/osmodel.mak
 
 export OS
+BUILD=release
```

Comment:
> Because you need to find the DMD binary somehow and as Makefile variable at least the user can overwrite it...

I mean, is there s reason for preferring `release` over `debug`? If not I think it would be better to stick with `debug`.